### DISCUSSION TITLE
Table of trilium notes

### DISCUSSION THREAD

**Discussion by pandasstart** [TriliumNext/Trilium#4346] [Ideas]
I want to set the color and size of text in batches in the table, but I cannot. I can only modify one cell, which is not very user-friendly for the user experience.

**Comment by jwhonce** [comment]
I can click a cell, then shift+click within a row or column to edit all the intermediate cells inclusive of the two selected. Is there something more you are looking for?

I am running the 0.60.4 client on fedora linux.

**Reply by pandasstart** [reply]
首先非常感谢您的解答，但是可能还没有解决我的需求，我希望是可以对单元格内的文字做调整，比如修改字体的大小（我说的是批量对多个单元格或者是表格的所有单元格里面的文字做调整），我现在只能对单个单元格里面的文字做调整。
